templates/c2s_local_dev/README.md

```diff
@@ -0,0 +1,9 @@
+# C2S docker-compose file for local development
+
+1. Clone this repo locally `git https://github.com/nhsuk/nhsuk-rancher-templates.git`
```

Comment:
You might want to add `clone` to that command e.g. `git clone https://github.com/nhsuk/nhsuk-rancher-templates.git`